resources/prometheus/prometheus-rules.yaml

```diff
@@ -603,30 +603,30 @@ spec:
             cpu_resource_limits:strictly_worker_nodes:by_availability_zone:sum
             /
             availability_zone:strictly_worker_nodes:allocatable_cpu
-        - alert: WorkerNodesByAZMemoryQuotaOverCommit
-          expr: availability_zone:strictly_worker_nodes:memory_request_ratio > 0.8
+        - alert: WorkerNodesMemoryQuotaOverCommit
+          expr: sum(availability_zone:strictly_worker_nodes:memory_request_ratio) > 0.8
```

Comment:
Does this do the right thing? Shouldn't it be `avg`?

Reply:
or rather max..

Reply:
No you're right